census/src/main/java/io/grpc/census/CensusStatsModule.java

```diff
@@ -707,4 +650,147 @@ public void onClose(Status status, Metadata trailers) {
       };
     }
   }
+
+  @VisibleForTesting
+  static final class CallAttemptsTracerFactory extends
```

Comment:
Why was this moved within the file? It makes the diff hard to read and it took me a while to realize that it was just moved within the file.

Reply:
Sorry, this was modified from the first POC PR, which reimplemented the factory and the factory is instantiated for every attempt. Will move it back.